lutris/util/system.py

```diff
@@ -451,3 +451,21 @@ def is_running(process):
         if re.search(process, x):
             return True
     return False
+
+
+def check_ntfs(path):
+    # import here to avoid cyclic import with lutris.gui.dialog
+    from lutris.gui.dialogs import ErrorDialog
+
+    def display_ntfs_warning():
+        ErrorDialog(
+            "<b>Using NTFS partition causes major issues!</b>\n")
+    out = None
+
+    if path:
+        out = subprocess.check_output(["df", "-Th", path]).decode("UTF-8")
```

Comment:
Would this work instead: https://docs.python.org/3.7/library/os.html#os.fstatvfs ?

Reply:
I'm not sure how to use it the output is cryptic here is an example `os.statvfs_result(f_bsize=4096, f_frsize=4096, f_blocks=0, f_bfree=0, f_bavail=0, f_files=0, f_ffree=0, f_favail=0, f_flag=4106, f_namemax=255)
`

Reply:
weird it gives a lot of info about the file system but not what it is or I overlooked something

Reply:
According to os.statvfs the support for filesystem ID field f_fsid was only added in Python 3.7.

Reply:
I compared os.statvfs out in ntfs and ext4 unfortunately nothing strikes out

Reply:
weird I have python 3.7.1 and I still don't see f_fsid in the output

Reply:
either way I guess that's too recent for what lutris targets